dotcom-rendering/src/model/enhanceTreats.ts

```diff
@@ -31,25 +31,25 @@ const PLATFORM_TREATS: TreatType[] = [
 		altText: 'The White House',
 		pageId: 'us',
 	},
-	{
-		links: [
-			{
-				linkTo: '/tv-and-radio/ng-interactive/2022/aug/01/whats-on-netflix-and-amazon-this-month-august',
-				text: "What's on Netflix & Amazon this month",
-			},
-			{
-				linkTo: '/books/ng-interactive/2022/aug/19/this-months-best-paperbacks-jan-morris-katie-kitamura-and-more',
-				text: "This month's best paperbacks",
-			},
-		],
-		theme: ArticlePillar.Culture,
-		containerTitle: 'Culture',
-		editionId: 'UK',
-		imageUrl:
-			'https://uploads.guim.co.uk/2022/08/19/culture-nugget-august.png',
-		altText: "What's on Netflix and Amazon this month",
-		pageId: 'uk',
-	},
+	// {
```

Comment:
Might want to leave a comment about why this is commented out! That or just remove it entirely, we can always open a revert PR if and when its back!

Reply:
Good point, I've removed it